### ISSUE TITLE
Tests timeout and not run

### ISSUE THREAD

**Issue by bbarker** [joelburget/easytest#21]
I have a test suite that has 4 individual tests (scopes); the 4th test takes a while to run, and actually times out and does not run at all (only 3 tests are reported, and all seem to succeed). However, the 4th test generates a cache on disk, and due my guess that some sort of timeout is occurring, it is in a bad state, so when I run the same test suite again, the 4th test *does* run, and it fails:

```
smaXtec.processEntireFiles EXCEPTION: Prelude.read: no parse                                                                                                                       
FAILED smaXtec.processEntireFiles  
```

There's some guess work going on here, I'll see if I can find more information; so far it seems not quite deterministic (though I feel that it should be).

**Comment by joelburget** [comment]
Interesting -- what timeout does the fourth test hit? That's not an easytest thing is it?

**Comment by joelburget** [comment]
This is probably fixed by the 0.3 release. Please reopen if not.
